Dockerfile

```diff
@@ -1,11 +1,10 @@
 FROM debian:stable-slim
 
-RUN apt-get update && apt-get -uy upgrade
-RUN apt-get -y install ca-certificates && update-ca-certificates
+RUN apt-get -qq update && apt-get -yyqq install ca-certificates && apt-get clean
```

Comment:
What is the reason for this change?  I see it removes `apt-get upgrade` adds `apt-get clean`.

Reply:
apt-get clean - clean up layer (speeds up build)
apt-get upgrade - doesn't affect layer product (i.e. /etc/ssl/certs/ca-certificates.crt) in any way